### PR TITLE
Updated fetch import to fix server error

### PR DESCRIPTION
**Pull request by Mukul-code-rep**
I was working with avalanchejs and tried to automate my code using rest api created from express and morgan. When I tried to run it using heroku, it gave reference errors and upon further research, I found a fix and think that it should be put on the package.

### CODE REVIEW

**Comment by barbas-gen** [comment]
Running into same issue

**Comment by LoganDiliberto-Crypto** [comment]
Same issue here and this seemed to work

File: src/utils/fetchadapter.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { AxiosRequestConfig, AxiosResponse, AxiosError } from "axios"
+const f = require('node-fetch');
```
Comment (gergelylovas) [inline src/utils/fetchadapter.ts:2]: `node-fetch` is not amongst the dependencies of avalancheJS.

**Review by gergelylovas — CHANGES_REQUESTED** [review]
These changes are not the way to make avalancheJS work with Node<18.

This only works because of the import of `node-fetch` which polyfills the missing `fetch` API and not because of the changes of `Request` => `f.Request` and `Headers` => `f.Headers`

The import of `node-fetch` and thus polyfilling the `fetch` API should happen in the consumer project, prior the avalancheJS imports. Which will make AvalancheJS work in Node <18 runtimes.

**Pull request closed without merging**